content/en/blogs/understand-requests-and-limits-in-kubernetes.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Dive Deep into Resource Requests and Limits in Kubernetes'
+title: 'Kubernetes Resource Requests and Limits'
```

Comment:
How about `Kubernetes Resource Requests and Limits Deep Dive`?

Reply:
Sounds great